pkg/debug/checkups/power_windows.go

```diff
@@ -0,0 +1,64 @@
+//go:build windows
+// +build windows
```

Comment:
This is fine, but we don't need to build tag -- can switch in the `Name()`. Mostly depends what feels easier

Reply:
We do have to use the build tag for this, unfortunately -- before I separated them out and added the tag: https://github.com/kolide/launcher/actions/runs/5813681186/job/15761773747